docs/tutorial/getting-started.md

```diff
@@ -0,0 +1,78 @@
+# Deploy the Wazuh Server charm for the first time
+
+## What you’ll do
+- Deploy the Wazuh Server charm.
+- Integrate with the Wazuh Indexer charm.
+- Integrate with the elf Signed X.509 Certificates charm.
+- Integrate with the Traefik charm.
+
+## Requirements
+
+* A working station, e.g., a laptop, with amd64 architecture.
+* Juju 3 installed and bootstrapped to a MicroK8s and to an LXD controller. You can accomplish
+this process by using a [Multipass](https://multipass.run/) VM as outlined in this guide: [Set up / Tear down your test environment](https://juju.is/docs/juju/set-up--tear-down-your-test-environment)
+
+:warning: When using a Multipass VM, make sure to replace IP addresses with the
+VM IP in steps that assume you're running locally. To get the IP address of the
+Multipass instance run ```multipass info my-juju-vm```.
+
+## Set up a Tutorial Model
+
+To manage resources effectively and to separate this tutorial's workload from
+your usual work, create a new model using the following command.
+
+```bash
+juju add-model wazuh-tutorial
+```
+
+## Deploy the Wazuh Server charm
+Synapse requires connections to Wazuh indexer, Traefik and a charm implementing the TLS certificates relation,
+for this tutorial we will be using Self Signed X.509 Certificates. Deploy all these charm applications.
+
+### Deploy and integrate the charms
+
+```bash
+juju deploy wazuh-server
+juju deploy self-signed-certificates
+juju deploy traefik-k8s --trust
+```
+For instructions to deploy the Wazuh Indexer, check [its documentation](https://charmhub.io/wazuh-indexer).
```

Comment:
You may want to move this into the "Requirements" section, and specify that Indexer will need to be deployed in a machine model and in a different controller